### PR TITLE
[#3] Add aio Dockerfile

### LINKED ISSUE #3: Do not use AIO to build testcontainer image
AIO will be removed from neofs-node repo, so use neofs-node, neofs-ir and neofs-cli images separately.

### PR DESCRIPTION
**Pull request by KirillovDenis**
closes #3 

Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+ARG HUB_IMAGE=nspccdev/neofs
+ARG HUB_TAG=latest
+
+FROM ${HUB_IMAGE}-cli:${HUB_TAG} as neofs-cli
+FROM ${HUB_IMAGE}-ir:${HUB_TAG} as neofs-ir
+FROM ${HUB_IMAGE}-storage:${HUB_TAG} as neofs-storage
+#FROM ${HUB_IMAGE}-adm:${HUB_TAG} as neofs-adm
```
Comment (alexvanin) [inline Dockerfile:7]: It should be available now, check DockerHub.

**Review by alexvanin — COMMENTED** [review]
Can you also remove aio from neofs-node repo then?

File: testcontainer/init-testcontainer.sh [hunk 2]
```diff
@@ -9,6 +9,26 @@ do
...
+set -m
+/bin/neofs-node --config /config/config-sn.yaml &
+
+while [[ -z "$(/bin/neofs-cli control healthcheck -r localhost:16513 --binary-key /config/wallet-sn.key | grep 'Health status: READY')" ]];
+do
+  sleep 1;
+done
+
+echo "aio container started"
+fg
```
Comment (alexvanin) [inline testcontainer/init-testcontainer.sh:34]: Is `fg` required?

**Review by alexvanin — APPROVED** [review]

File: testcontainer/init-testcontainer.sh (hunk 2 above)
Comment (KirillovDenis) [reply testcontainer/init-testcontainer.sh:34]: Yes. Otherwise container will exit

**Review by realloc — APPROVED** [review]

**Pull request merged**